Wazuh Cloud: Documentation for the new pricing model (#6258)
wazuh/wazuh-documentation: source/cloud-service/glossary.rst

The Cloud ID is a unique ID for your environment on Wazuh Cloud. It is used for multiple purposes, such as Wazuh WUI access or the agent registration process.

.. _cloud_glossary_environment:

Environment
-----------

An environment is a deployment that contains all the Wazuh components ready to use and running on Wazuh Cloud.

.. _cloud_glossary_archived_data:

> Archived data (formerly known as cold storage)

Reviewer: ```suggestion
Archived data
```

----------------------------------------------

> It is the data containing the output generated by Wazuh, such as alerts or archives. It is an AWS S3 bucket to store your logs for a longer time and meet compliance requirements.

Reviewer: ```suggestion
Formerly known as cold storage, it's the data containing the output generated by Wazuh, such as alerts and archives. It's an AWS S3 bucket to store your logs for a longer time and meet compliance requirements.
```

> 

Reviewer: ```suggestion
```

.. _cloud_glossary_hot_storage:

> Indexed data (formerly known as hot storage) 

Reviewer: ```suggestion
Indexed data
```

--------------------------------------------

> It is the data available on the Wazuh WUI corresponding to the information indexed by Wazuh. This information is available as soon as Wazuh ingests and indexes the events sent by the agents, making the data searchable and analyzable.

Reviewer: ```suggestion
Formerly known as hot storage, it's the data available on the Wazuh dashboard corresponding to the information indexed by Wazuh. This information is available as soon as Wazuh ingests and indexes the events sent by the agents, making the data searchable and analyzable.
```

Indexed data is calculated using the primary shards of ``wazuh-*`` indices.

.. _cloud_glossary_tier:

Tier
----

> The concept of a tier, which represents the size limitation, in bytes, of the indexed data (formerly known as hot storage), is no longer used. It has been replaced by the *indexed data size* setting.

Reviewer: ```suggestion
The term *tier* is no longer used. It has been replaced by the *indexed data size* setting. It represented the size limitation, in bytes, of the indexed data, formerly known as *hot storage*.
```

.. _cloud_glossary_setting:

Setting
-------

> A setting in the context of Wazuh Cloud refers to each of the configuration options available for a cloud environment. These settings define the limitations, capabilities, and pricing of the environment.

Reviewer: ```suggestion
In the context of Wazuh Cloud, a setting  refers to each configuration option available for a cloud environment. These settings determine the limitations, functionalities, and pricing of an environment.
```
